pyproject.toml

```diff
@@ -96,6 +96,7 @@ develop = [
     "pytest-asyncio==0.19.0",
     "pytest-httpserver==1.0.5",
     "tox==3.25.0",
+    "nox==2022.11.21",
```

Comment:
If we are introducing `nox`, can we sunset `tox`?

Reply:
Not yet, we still run IT and rally-tracks-compat through tox. It will definitely be sunset in a few PRs.